.github/workflows/gradle.yml

```diff
@@ -16,10 +16,10 @@ jobs:
     environment: BRouter
     steps:
     - uses: actions/checkout@v4
-    - name: Set up JDK 11
+    - name: Set up JDK 17
```

Comment:
One remark only, the BRouter server still uses JDK 11. 
Therefore, the output lib cannot be used directly.

Reply:
If we want to compile for older Java releases we should [target a specific java version](https://docs.gradle.org/current/userguide/building_java_projects.html#sec:java_cross_compilation). It seems reasonable to target Java 11 but use JDK 17 to build.